src/components/Note/Note.scss

```diff
@@ -170,12 +165,13 @@ $note__author-image-size: 26px;
   flex-grow: 0;
   height: 28px;
   padding: 0 $margin--default 0 0;
+  background: $color-white;
+  border-radius: 21px;
+  padding-left: 2px;
```

Comment:
`padding` and `padding-left` can be merged here

Reply:
fixed it, thanks